release/src/version-helpers.ts

```diff
@@ -76,6 +76,10 @@ export const getMajorVersion = (versionString: string) =>
     .replace(/-rc\d+/i, "")
     .split(".")[0];
 
+export const isReleaseBranch = (branchName: string) => {
+  return branchName.includes("release-x");
```

Comment:
nit: use startsWith to avoid false positives

```suggestion
  return branchName.startsWith("release-x");
```